integrations/sentry/main.py

```diff
@@ -32,44 +39,73 @@ async def add_tags_to_issue(
     return {**issue, "__tags": tags}
 
 
+async def add_tags_to_project(
+    project: dict[str, Any], sentry_client: SentryClient
+) -> list[dict[str, Any]]:
+    tags = await sentry_client.get_project_tags(project["slug"])
+    return [{**project, "__tags": tag} for tag in tags]
+
+
+async def _get_paginated_project_slugs() -> ASYNC_GENERATOR_RESYNC_TYPE:
+    sentry_client = init_client()
+    async for projects in sentry_client.get_paginated_projects():
+        project_slugs = [project["slug"] for project in projects]
+        yield project_slugs
+
+
+async def _get_paginated_issues(project_slug: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    sentry_client = init_client()
+    async for issues in sentry_client.get_paginated_issues(project_slug):
+        yield issues
+
```

Comment:
are those redundant and we can instead just use `sentry_client .get_paginated_issues?`